_posts/2016-02-08-terminal-commands.md

```diff
@@ -0,0 +1,233 @@
+# Importance of using the terminal?
+
+Under multiple platforms the Operating System has the GUIs (graphical user interfaces), where you can as a user point, click and drag on multiple components of the OS, which makes the user experience extremely easy and friendly. 
+
+The traditional Unix environment are based into CLI (command line interface), where you as a user have the necessity to type commands to tell the computer what to do. Believe it or not, that is faster and more powerful to handle the Operating System.
```

Comment:
@marticam : remove "the" before "traditional" and make "environment" plural like "environments"
I'm not sure about the "into CLI" part, @ssjuampi23 any thoughts?